pkg/rancher-desktop/components/Nav.vue

```diff
@@ -115,6 +121,9 @@ export default {
         },
       };
     },
+    isOnWhitelist(name: string): boolean {
```

Comment:
Rather than `isOnWhitelist`, we should call this `isKnownMonochrome` or something like that.  `whitelist` by itself doesn't say what we're whitelisting _for_.

Reply:
Also don't use the term "whitelist"; use "allowlist" and "blocklist" (or "denylist").